tests/test_calibration.py

```diff
@@ -375,6 +375,7 @@
 
 
 @pytest.mark.slow
+@pytest.mark.xfail(reason="SpotPy Calibration is not entirely stable.")
```

Comment:
This is one of the tests that "sometimes" fails, regardless of build environment. There are a handful of tests that are not threadsafe, but because they kill the pytest workers when they fail, there's no way to mark them as xfailable.

Reply:
I wouldn't be surprised if the failures are due to the longstanding libnetcdf issues with asynchronous read of NetCDF4 files.

Reply:
Nevermind, it seems this test can also fail with a crash as well:
![image](https://github.com/CSHS-CWRA/RavenPy/assets/10819524/97fd9af3-385d-4da8-9f86-ae72142afa42)